docs/file-formats.rst

```diff
@@ -42,6 +42,11 @@ and building specific layers via the :ref:`priority indexes <priority-indexes>`
 :ref:`package indexes <package-indexes>` layer settings respectively. The :uv-config:`sources`
 field in the common configuration is also restricted to only referring to named indexes.
 
+If any indexes require authentication to access, this must be configured via the ``uv``'s
```

Comment:
Typo here (redundant `the`), but I'm about to compile the 0.8.0b1 release notes, so I'll fix it there rather than triggering a fresh CI build here.